client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx

```diff
@@ -377,6 +439,32 @@ export const UserAddCodeHostsPage: React.FunctionComponent<UserAddCodeHostsPageP
         </Alert>
     )
 
+    const getOutageMessage = (servicesDown: string[]): JSX.Element => (
+        <Alert className="my-3" key={servicesDown[0]} variant="warning">
+            {servicesDown?.map(svc => (
+                <div key={svc}>
+                    <h4 className="align-middle mb-1">We're having trouble connecting to {svc} </h4>
+                    <p className="align-middle mb-0">
+                        <span className="align-middle">Before continuing, verify that</span> {svc}
```

Comment:
```suggestion
                        <span className="align-middle">Verify that</span> {svc}
```

There isn't really a flow "being continued" here, so simplifying the message.

Reply:
Fixed
<img width="647" alt="Screen Shot 2022-03-30 at 10 41 20 AM" src="https://user-images.githubusercontent.com/1552863/160901112-8a336091-7835-44dc-9b89-6baf00f1d444.png">